STABLE_RELEASES.md

```diff
@@ -0,0 +1,147 @@
+potential new candidates as well as the potential for existing code-owners to
+exit or "pass on the torch". This private meeting is to be a held as a
+phone/video meeting. Subsequently at the end of the meeting, one of the existing
+code owners should open a PR modifying the `CODEOWNERS` file. The other code
+owners should then all approve this PR to publicly display their support.
+
+Only if unanimous consensus is reached among all the existing code-owners will
+an invitation be extended to a new potential-member. Likewise, when an existing
+member is suggested to be removed/or have their privileges reduced, the member
+in question must agree on the decision for their removal or else no action
+should be taken. If however, a code-owner is verifiably shown to intentionally
+have had acted maliciously or grossly negligent, code-owner privileges may be
+stripped with no prior warning or consent from the member in question.
+
+Earning this privilege should be considered to be no small feat and is by no
+means guaranteed by any quantifiable metric. It is a symbol of great trust of
+the community of this project.
+
+# 0.37
+
+The `0.37.x` release series will continue receiving bugfixes until the Cosmos Hub
+migrates to a newer release of the Cosmos-SDK.
+
+# 0.38
+
+Discontinued since **0.39.0** is published.
+
+# 0.39 - The Launchpad Release Series
+
+The **0.39 Launchpad** release series will be supported until 6 months after **0.40.0** is published. A fairly strict **bugfix-only** rule applies to pull requests that are requested to be included into a stable point-release.
+
+## What pull requests will be included in stable point-releases
+
+Pull requests that fix bugs that fall in the following categories do not require a **Stable Release Exception** to be granted to be included in a stable point-release:
+
+ * **Severe regressions**.
+ * Bugs that may cause **client applications** to be **largely unusable**.
+ * Bugs that may cause **state corruption or data loss**.
+ * Bugs that may directly or indirectly cause a **security vulnerability**.
+
+## What pull requests will NOT be automatically included in stable point-releases
+
+As rule of thumb, the following changes will **NOT** be automatically accepted into stable point-releases:
+
+ * **State machine changes**.
+ * **Client application's code-breaking changes**, i.e. changes that prevent client applications to build without modifications to the client's source code.
+ 
+ In some circumstances, PRs that don't meet the aforementioned criteria might be raised and asked to be granted a *Stable Release Exception*.
+ 
+ ## Stable Release Exception - Procedure
+ 
+1. Check that the bug is either fixed or not reproducible in `master`. It is, in general, not appropriate to release bugfixes for stable releases without first testing them in `master`. Please apply the label [0.39 (Launchpad)](https://github.com/cosmos/cosmos-sdk/labels/0.39%20LTS%20%28Launchpad%29) to the issue.
```

Comment:
this doc is very specific to the current releases. SHould it be? should we have this be generalized then a seperate doc that displays supported versions?

Reply:
Yes I made it specific to Launchpad, the reason being that `0.37` and `0.39` will be both supported in different ways/in compliance with difference policies (and I'm taking care of Launchpad's support policy only). Nevertheless, I could make the Stable Release policy and procedure generic and making clear in the rest of the document that only Launchpad stable relases will be managed in compliance with such policy. How does that sound?